src/integTest/kotlin/io/snyk/plugin/ui/toolwindow/SnykToolWindowPanelIntegTest.kt

```diff
@@ -41,17 +43,23 @@ class SnykToolWindowPanelIntegTest : HeavyPlatformTestCase() {
         unmockkAll()
         resetSettings(project)
         setupDummyCliFile()
+        // restore modified Registry value
+        isIacEnabledRegistryValue.setValue(isIacEnabledDefaultValue)
+        unmockkStatic("io.snyk.plugin.UtilsKt")
     }
 
     override fun tearDown() {
         unmockkAll()
         resetSettings(project)
         removeDummyCliFile()
+        // restore modified Registry value
+        isIacEnabledRegistryValue.setValue(isIacEnabledDefaultValue)
+        unmockkStatic("io.snyk.plugin.UtilsKt")
```

Comment:
Same here, why do you unmock utils explicitly and not just call unmockAll() at the end of the method?